scripts/README.md

```diff
@@ -41,3 +41,68 @@ Usage:
    $ ./get-image-changes.sh VERSION-TO-UPDATE > changes
    $ cat changes
 ```
+
+## `kernel_monitor/kernel_monitor.py`
+
+This script fetches the latest kernel information and looks for new LTS releases.
+If it finds a new LTS release it sends an email to the selected recipients.
+
+Requirements:
+
+* python3
+* python dependencies (see the [requirements file](https://github.com/clearcontainers/packaging/scripts/kernel_monitor/requirements))
+* pip
+* postfix
+
+Usage:
+
+NOTE: The following commands can be executed if you have a Fedora 27 system that meets all requirements,
+and a postfix server running on 127.0.0.1 or localhost.
+
+Install python dependencies:
+
+NOTE: It is strongly recommended to run this script in a Python virtual environment with `virtualenv`.
+```
+$ cd kernel_monitor
+$ virtualenv km
+$ source km/bin/activate
+$ pip install -r requirements
+```
+
+Set distro_certs variable to: `/etc/ssl/certs/ca-bundle.crt`:
+
+NOTE: This path might change based on the Linux distribution you are using (see kernel_monitor.py).
+```
+sed -i "s/distro_certs = .*/distro_certs = \"\/etc\/ssl\/certs\/ca-bundle.crt\"/" kernel_monitor.py
+```
+
+Execute the following script:
+```
+$ chmod +x kernel_monitor.py
+$ ./kernel_monitor.py
+```
+
+This sends an email to `root@localhost` if there is a new LTS release and log
```

Comment:
I'd remove `root@localhost` and the logfile name here (which I think is now incorrect?) and explain that running, `./kernel_monitor.py --help` will show where the emails will be sent and which logfile will be used.